airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/FutureUtils.java

```diff
@@ -12,16 +12,18 @@
 
 public class FutureUtils {
 
+  private static final int DEFAULT_TD_THREAD_COUNT = 8;
+
   /**
-   * Allow for configuring the number of typing and deduping threads via an enviornment variable in
+   * Allow for configuring the number of typing and deduping threads via an environment variable in
    * the destination container.
    *
    * @return the number of threads to use in the typing and deduping pool
    */
-  public static int countOfTypingDedupingThreads(final int defaultThreads) {
+  public static int getCountOfTypeAndDedupeThreads() {
     return Optional.ofNullable(System.getenv("TD_THREADS"))
         .map(Integer::valueOf)
-        .orElse(defaultThreads);
+        .orElse(DEFAULT_TD_THREAD_COUNT);
```

Comment:
Curious, why isn't this defined in the config?

Reply:
Just because its something that we want the platform to be able to configure rather than the individual destinations - wanting to keep our "advanced" option set small

Reply:
Ahh, got it, thanks! Makes sense.